Marlin/src/inc/SanityCheck.h

```diff
@@ -602,6 +602,12 @@ static_assert(X_MAX_LENGTH >= X_BED_SIZE && Y_MAX_LENGTH >= Y_BED_SIZE,
     #error "EXTRUDERS must be 1 with HEATERS_PARALLEL."
   #endif
 
+  #if ENABLED(SINGLENOZZLE)
+    #ifndef(SINGLENOZZLE_FILAMENTLENGTH)
+      #define SINGLENOZZLE_FILAMENTLENGTH 0.0;
+    #endif
+  #endif
+
```

Comment:
This belongs in one of the Conditionals files, not here.

Reply:
Moved to conditionals_post